test/tpm2-totp.sh

```diff
@@ -24,12 +24,12 @@ function prereq()
 {
     if [ -f NVChip ]; then
         echo "There is a leftover file NVChip in the test directory."
-        return 1
+        return 99 #XFAIL
```

Comment:
Shouldn't this be `ERROR` instead of `XFAIL` as in [test/test/libtpm2-totp.sh](https://github.com/tpm2-software/tpm2-totp/pull/4/files#diff-66b454918f2629e873f6fe27ab5c1d09)? The test doesn't appear to be in `XFAIL_TESTS`, so the test suite should fail if the test results in a hard error.

Reply:
Done